hw/dv/tools/testplans/enable_reg_testplan.hjson

```diff
@@ -0,0 +1,20 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+{
+  entries: [
+    {
+      name: enable_reg
+      desc: '''
+            The CSR test sequences will read and write accessible CSRs including the enable
+            registers and their locked registers. The RAL model supports predicting the correct
+            value of the locked registers based on their enable registers.
+            '''
+      milestone: V2
+      tests: ["{name}{intf}_csr_hw_reset",
```

Comment:
Are all of these tests verifying the lock register behavior?

Reply:
Thanks for the feedback, at first I thought `csr_hw_reset` has loops to issue multiple reset. I will delete the `csr_hw_reset`. I think the rest of the sequences do test the enable registers. 
I added `csr_bit_bash` because writing 0 to the enable register does not clear the lock status, the only way to clear it is to issue reset.

Reply:
SG, thanks.